RELEASE.md

```diff
@@ -1,3 +1,48 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* tf.data optimizations. Users can now represent, get and set options of a tf.data pipeline.
```

Comment:
I would replace this with:

tf.data users can now use `tf.data.experimental.AUTOTUNE` for the `num_parallel_calls` argument of `tf.data.Dataset.interleave`, `tf.data.Dataset.map`, and `tf.data.experimental.map_and_batch` to dynamically tune the level of parallelism at runtime.

and move the following to `Bug Fixes and Other Changes` under tf.data

tf.data users can now represent, get, and set options of TensorFlow input pipelines using `tf.data.Options()`, `tf.data.Dataset.options()`, and `tf.data.Dataset.with_options()` respectively.

Reply:
Done.